__tests__/app.test.js

```diff
@@ -93,4 +93,34 @@ describe("CORE: GET - /api/articles/:article_id", () => {
         expect(body).toEqual({ Error: 'ID Does Not Exist' })
       })
     })
-  })
\ No newline at end of file
+  })
+
+
+  describe("CORE: GET /api/articles/:comment_id/comments", () => {
+    test("200: Endpoint should contain all comments about specific article", () => {
+      return request(app)
+        .get("/api/articles/1/comments")
+        .expect(200)
+        .then(({ body }) => {
+          console.log(body)
+          body.forEach((comment) => {
+            expect(comment).toHaveProperty("comment_id", expect.any(Number));
+            expect(comment).toHaveProperty("votes", expect.any(Number)); 
+            expect(comment).toHaveProperty("created_at", expect.any(String)); 
+            expect(comment).toHaveProperty("author", expect.any(String)); 
+            expect(comment).toHaveProperty("body", expect.any(String)); 
+            expect(comment).toHaveProperty("article_id", (1)); 
+          });
+        });
+    });
+    test("200: Endpoint should be ordered by creation date (descending)", () => {
+      return request(app)
+        .get("/api/articles/1/comments")
+        .expect(200)
+        .then(({ body }) => {
+          
+          expect(body).toBeSortedBy('created_at', {descending : true})
+          });
+        });
+        //write error case for 0 results : 404
```

Comment:
Incomplete tests for this ticket. Please add tests for the following

Status 200, valid ID, but has no comments responds with an empty array of comments
Status 400, invalid ID, e.g. string of "not-an-id"
Status 404, non existent ID, e.g. 0 or 9999